### PR TITLE
Add formulas & factorial packages

### PR DESCRIPTION
**Pull request by Jarvis2001**
Added package-meta for formulas and factorial

### CODE REVIEW

**Comment by Jarvis2001** [comment]
Function content added in factorial.js and formulas.js since the calculation is based on BODMAS package sample calculation functions(mul, div, sqr, pow) are inserted instead of actual BODMAS(yet to be developed) functions.

**Comment by Jarvis2001** [comment]
Added return statement in formulas and factorial

File: packages/factorial/factorial.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// import * from bodmas 
+export default function factorial() {
```
Comment (kasipavankumar) [inline packages/factorial/factorial.js]: Please change this to `module.exports` as we cannot use `export default` here.
```js
module.exports = function factorial() {}
```

File: packages/formulas/formulas.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
+
+//Phy
+//Speed/Velocity
+export function spd(){
```
Comment (kasipavankumar) [inline packages/formulas/formulas.js]: Remove `export` from all the functions & at the end of the file add the following code:
```js
module.exports = {
  spd,
  acc,
  momen,
  // Add all the function names here
}
```

**Review by kasipavankumar — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Review by kasipavankumar — APPROVED** [review]
LGTM!

**Pull request merged**